### ISSUE TITLE
ElvUI Lua Error "QuestWatch.lua" on startup Classic SoD. 

### ISSUE THREAD

**Issue by 123ebeast** [tukui-org/ElvUI#1289]
### Troubleshooting Steps

When logging on and choosing any character using ElvUI addon, I get presented with a Lua Error after Blizzard updated their client some days ago. I have followed the steps for troubleshooting and attached is the screenshot. 
![ElvUI_Error_QuestWatch](https://github.com/user-attachments/assets/ba58aede-e4f2-43c1-b01d-22152caa4281)

### Describe the expected behavior and what actually happened?

When logging on and choosing any character using ElvUI addon, I get presented with a Lua Error after Blizzard updated their client some days ago.

### Reproducing the issue.

Just logon to any character with ElvUI enabled and the message will present itself. 

### Lua Errors

```shell
`Message: ...ce/AddOns/ElvUI/Core/Modules/Blizzard/QuestWatch.lua:32: attempt to index local 'QuestTimerFrame' (a nil value)
Time: Tue Aug  6 13:16:25 2024
Count: 1
Stack: ...ce/AddOns/ElvUI/Core/Modules/Blizzard/QuestWatch.lua:32: attempt to index local 'QuestTimerFrame' (a nil value)
[string "@Interface/AddOns/ElvUI/Core/General/Core.lua"]:1822: in function `CallLoadFunc'
[string "@Interface/AddOns/ElvUI/Core/General/Core.lua"]:1837: in function `CallLoadedModule'
[string "@Interface/AddOns/ElvUI/Core/General/Core.lua"]:1863: in function `InitializeModules'
[string "@Interface/AddOns/ElvUI/Core/General/Core.lua"]:1975: in function `Initialize'
[string "@Interface/AddOns/ElvUI/Core/init.lua"]:302: in function <Interface/AddOns/ElvUI/Core/init.lua:301>
[string "=[C]"]: ?
[string "@Interface/AddOns/Atlas/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:66: in function <...face/AddOns/Atlas/Libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Interface/AddOns/Atlas/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:523: in function `EnableAddon'
[string "@Interface/AddOns/Atlas/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:626: in function <...face/AddOns/Atlas/Libs/AceAddon-3.0/AceAddon-3.0.lua:611>

Locals: self = <table> {
 GetTimeInfo = <function> defined @Interface/AddOns/ElvUI/Core/General/Math.lua:354
 mylevel = 60
 Bags = <table> {
 }
 privateVars = <table> {
 }
 modules = <table> {
 }
 FormatMoney = <function> defined @Interface/AddOns/ElvUI/Core/General/Math.lua:411
 Config_OpenWindow = <function> defined @Interface/AddOns/ElvUI/Core/General/Config.lua:1029
 UpdateUnitFrames = <function> defined @Interface/AddOns/ElvUI/Core/General/Core.lua:1436
 SetMoversPositions = <function> defined @Interface/AddOns/ElvUI/Core/General/Movers.lua:453
 ScheduleTimer = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceTimer-3.0/AceTimer-3.0.lua:94
 PriestColors = <table> {
 }
 RemoveEmptySubTables = <function> defined @Interface/AddOns/ElvUI/Core/General/Core.lua:624
 Cooldown_Options = <function> defined @Interface/AddOns/ElvUI/Core/General/Cooldowns.lua:138
 Cooldown_UnbuggedTime = <function> defined @Interface/AddOns/ElvUI/Core/General/Cooldowns.lua:19
 Config_CreateFrame = <function> defined @Interface/AddOns/ElvUI/Core/General/Config.lua:827
 UIScale = <function> defined @Interface/AddOns/ElvUI/Core/General/PixelPerfect.lua:55
 db = <table> {
 }
 ScanTooltip_InventoryInfo = <function> defined @Interface/AddOns/ElvUI/Core/General/API.lua:955
 ScanTooltip_UnitInfo = <function> defined @Interface/AddOns/ElvUI/Core/General/API.lua:943
 Layout = <table> {
 }
 UnregisterEventForObject = <function> defined @Interface/AddOns/ElvUI/Core/General/Core.lua:1752
 Distributor = <table> {
 }
 DBConvertSL = <function> defined @Interface/AddOns/ElvUI/Core/General/Core.lua:1255
 MountText = <table> {
 }
 INCOMPATIBLE_ADDONS = <table> {
 }
 PositionGameMenuButton = <function> defined @Interface/AddOns/ElvUI/Core/General/API.lua:795
 UIFrameFade = <function> defined @Interface/AddOns/ElvUI/Core/General/Animation.lua:361
 TimeIndicatorColors = <table> {
 }
 StaticPopup_IsLastDisplayedFrame = <function> defined @Interface/AddOns/ElvUI/Core/General/StaticPopups.lua:486
 StaticPopup_CreateSecureButton = <function> defined @Interface/AddOns/ElvUI/Core/General/StaticPopups.lua:1103
 Truncate = <function> defined @Interface/AddOns/ElvUI/Core/General/Math.lua:142
 SetupReset = <function> defined @Interface/AddOns/ElvUI/Core/General/Install.lua:544
 ToggleMoveMode = <function> defined @Interface/AddOns/ElvUI/Core/General/Config.lua:70
 Dump = <function> defined @Interface/AddOns/ElvUI/Core/General/API.lua:474
 WorldMap = <table> {
 }
 Disable = <function> defined @Interface/AddOns/Atlas/Libs/AceAddon-3.0/AceAddon-3.0.lua:310
 TimeColors = <table> {
 }
 SlideOut = <function> defined @Interface/AddOns/ElvUI/Core/General/Animation.lua:296
 Unhook = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:417
 XPIsTrialMax = <function> defined @Interfa`
```

### Verification Steps

- [X] I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/main/CHANGELOG.md) and verified [ElvUI](https://tukui.org/elvui) is up to date by checking **/estatus**
- [X] I have verified my issue doesn't exist in the Default UI or the [Issue Tracker](https://github.com/tukui-org/ElvUI/issues).
- [ ] I have not read any of these steps, please close my issue when you see it.
- [X] I have done the Troubleshooting Steps and included the requested screenshots, which show the issue and status panel.

**Comment by 1Botanica1** [comment]
Thanks for the report. This has already been fixed and will be in next release of ElvUI

**Comment by 123ebeast** [comment]
> Thanks for the report. This has already been fixed and will be in next release of ElvUI

Thank you so much!

**Comment by kodewdle** [comment]
fixed in 13.73
